src/engraving/rendering/dev/chordlayout.cpp

```diff
@@ -1889,7 +1889,8 @@ void ChordLayout::layoutChords1(LayoutContext& ctx, Segment* segment, staff_idx_
                         upOffset = maxDownWidth + adjSpace;
                         if (downHooks) {
                             bool needsHookSpace = (ledgerOverlapBelow || ledgerOverlapAbove);
-                            double hookSpace = topDownNote->chord()->hook()->width();
+                            Hook* hook = topDownNote->chord()->hook();
+                            double hookSpace = hook ? hook->width() : 0.0;
```

Comment:
Would a similar change be potentially necessary in line 1929?

Reply:
Yeah definitely a good idea

Reply:
for master and 4.2.1?